msfdb

```diff
@@ -673,7 +673,7 @@ def persist_data_service
 end
 
 def clear_default_data_service
-  puts 'Clearing http web data service credentials in msfconsole'
+  puts 'Verifying msfconsole data service'
```

Comment:
The old message could suggest to the user that data is being deleted. We still want this message around for debugging user issues - i.e. a crash after this message suggests issues booting msfconsole. We'll go with a more generic verb of 'verify'